test/integration/basic_excluded_prefixes_test.go

```diff
@@ -42,12 +42,14 @@ func TestExcludePrefixCheck(t *testing.T) {
 		},
 	}, k8s.GetK8sNamespace())
 
+	defer nsmd_test_utils.FailLogger(k8s, nodes, t)
+
 	icmp := nsmd_test_utils.DeployICMP(k8s, nodes[0].Node, "icmp-responder-nse-1", defaultTimeout)
 
 	clientset, err := k8s.GetClientSet()
 	Expect(err).To(BeNil())
 
-	_, err = clientset.CoreV1().Pods(k8s.GetK8sNamespace()).Create(pods.NSCPod("nsc", nodes[0].Node,
+	_, err = clientset.CoreV1().Pods("default").Create(pods.NSCPod("nsc", nodes[0].Node,
```

Comment:
Hmm not sure? Why go back to default?

Reply:
I haven't changed it intentionally, seems like consequences of rebase. Will fix